### PR TITLE
added control for circular magnetic field (according to Moshe Shahar)

### PR DESCRIPTION
**Pull request by shaiTheKimhi**

### CODE REVIEW

File: Modules/Control/Control.py [hunk 1]
```diff
@@ -8,17 +8,25 @@
...
-        timer = (self.time-time.time()) % (90*60)
-        torque = get_angle_vector(get_angle(timer))
-        z = (field[1] * torque[0] - filed[0]*torque[1]) / (field[2]*torque[1]-field[1]*torque[2])
-        y = (-field[0] - z * field[2]) / field[1]
-        return np.array([1,y,z])
+        #TODO : add a secondary thread that changes calls setControl
+        
+
+    #TODO: add offset for meneuvering to certain locations on earth
+    def SetControl(self,mode=1,offset=[0,0]):
```
Comment (yotam180) [inline Modules/Control/Control.py]: Naming

**Review by yotam180 — COMMENTED** [review]
And just go through the whole file and apply operator spacing, please. If you wish, I can specify all the lines where I wish operators to be spaced.

File: Modules/Control/Control.py (hunk 1 above)
Comment (ron-popov) [reply Modules/Control/Control.py]: spacing

**Review by yotam180 — APPROVED** [review]
Yep

**Comment by yotam180** [comment]
This is definitely NOT a pull request I have approved, and you are contaminating our code-cycle, @shaiTheKimhi. I'm reverting the changes.

**Comment by shaiTheKimhi** [comment]
I remember you did approve it, after I have changed waht you asked me to

**Comment by shaiTheKimhi** [comment]
Just to specify the changes:
I added the spacing you required and I changed the names of the functions and moved constants to the config file

**Comment by yotam180** [comment]
I don't remember doing so.

**Pull request merged**